src/components/GlobalHeader/index.tsx

```diff
@@ -6,8 +6,16 @@ import RightContent from './RightContent'
 
 const styles = require('./index.less')
 
-export default function GlobalHeader2 (props) {
-  // eslint-disable
+interface GlobalHeader2Props {
+  onCollapse: Function
+  isMobile: boolean
+  collapsed: boolean
+  logo: string
+}
+
+export default function GlobalHeader2 (props: GlobalHeader2Props) {
```

Comment:
I think I forgot to rename that component (to remove `2`), sorry. Since you are editing this lines, could you remove `2` from name, please?

Reply:
done. 👍